tensorboard/components/tf_dashboard_common/tf-downloader.html

```diff
@@ -80,7 +72,7 @@
         _run: String,
         runs: Array,
         tag: String,
-        urlFn: Function,
+        urlFn: Function,  // (tag: string, run: string) => string (JSON URL)
```

Comment:
Maybe move this comment up one line and just slightly expand to state that the user should provide this to generate URLs for the selected run/tag combos.

Reply:
Sure.